src/chttpd/test/eunit/chttpd_util_test.erl

```diff
@@ -19,24 +19,24 @@
 
 setup() ->
     ok = config:set("httpd", "both_exist", "get_in_httpd", _Persist = false),
-    ok = config:set("chttpd", "both_exist", "get_in_chttpd", _Persist = false),
-    ok = config:set("httpd", "httpd_only", "true", _Persist = false),
-    ok = config:set("chttpd", "chttpd_only", "1", _Persist = false),
-    ok = config:set("couch_httpd_auth", "both_exist", "cha", _Persist = false),
-    ok = config:set("chttpd_auth", "both_exist", "ca", _Persist = false),
-    ok = config:set("couch_httpd_auth", "cha_only", "true", _Persist = false),
-    ok = config:set("chttpd_auth", "ca_only", "1", _Persist = false).
+    ok = config:set("chttpd", "both_exist", "get_in_chttpd", false),
```

Comment:
Why remove `_Persist = `?

Reply:
I ran `emilio` before and it gave a warning about `_Persist`.

Reply:
Changed back.